relay-event-normalization/src/normalize/ai_model_costs.rs

```diff
@@ -0,0 +1,176 @@
+//! This file maps AI model IDs to their respective costs.
+//! Adapted from <https://github.com/langchain-ai/langchain/blob/5b6d1a907df8a7af0f47c5ffd394b3673d9539e5/libs/community/langchain_community/callbacks/openai_info.py#L4>
+
+/// Cost is in US dollars per thousand tokens
+fn calculate_cost_per_1k_tokens(model_id: &str, for_completion: bool) -> Option<f64> {
+    match (model_id, for_completion) {
+        // GPT-4 input
+        ("gpt-4", false) => Some(0.03),
```

Comment:
Are all values smaller than 1? If yes, have we considered forwarding integers and updating the unit to cents, to avoid floats?

Reply:
Since it's a cost per thousand tokens, you end up with floating point math at multiplication time anyways - since prompts won't be even multiples of a thousand